### ISSUE TITLE
Zone and numeric_state schema errors

### ISSUE THREAD

**Issue by Guyohms** [keesschollaart81/vscode-home-assistant#443]
- Zone does not accept `person` domain as a device tracker.
```YAML
trigger:
  - platform: zone
    zone: zone.home
    entity_id: 
      - person.karine
```
This generates error : `String does not match the pattern of "^device_tracker\.(?!_)[\da-z_]+(?<!_)$".`

- Numeric_state does not accept `proximity` domain.
```YAML  
trigger:
  - platform: numeric_state
    entity_id: proximity.home_guyaume

```
This generates error : ` String does not match the pattern of "^sensor\.(?!_)[\da-z_]+(?<!_)\s?(?:,\s?sensor\.(?!_)[\da-z_]+(?<!_))*$".
`

**Comment by frenck** [comment]
The numeric_state has been fixed in #442 

Will take a look at the zone trigger, thanks for reporting 👍

**Comment by frenck** [comment]
Zones fixed in #444
